docs/hippocampus_model.md

```diff
@@ -0,0 +1,35 @@
+# Trained model for hippocampal segmentation
+## Purpose
+This documentation describes how to use our pre-trained model to segment the left and right hippocampi from brain MRI scans. The model was trained on 998 MRI scan + segmentation pairs from the [ADNI](https://adni.loni.usc.edu/) dataset. This data is publicly available via their website, but users must sign a Data Use Agreement in order to gain access. We do not provide access to the data. The following description assumes the user has their own dataset to evaluate/ retrain the model on.
+
+## Terms of use
+Please note that this model is intended for research purposes only. You are responsible for the performance, the necessary testing, and if needed any regulatory clearance for any of the models produced by this toolbox.
+
+---
+
+## Usage
+The following instructions assume you have completed the preceding setup steps in the [InnerEye README](https://github.com/microsoft/InnerEye-DeepLearning/), in particular, [Setting up Azure Machine Learning](https://github.com/microsoft/InnerEye-DeepLearning/blob/main/docs/setting_up_aml.md).
+
+### Create an Azure ML Dataset
+To evaluate this model on your own data, you will first need to register an [Azure ML Dataset](https://docs.microsoft.com/en-us/azure/machine-learning/v1/how-to-create-register-datasets). You can follow the instructions in the InnerEye repo for [creating datasets](https://github.com/microsoft/InnerEye-DeepLearning/blob/main/docs/creating_dataset.md) in order to do this.
+
+## Downloading the model
+The saved weights from the trained Hippocampus model can be downloaded along with the source code used to train it from [our github releases page](https://github.com/microsoft/hi-ml/releases).
```

Comment:
Is the plan to update the URL once the model has been released?

Reply:
Yes that's the plan. There will be the overhead then that we will have to update the link any time we do a new release, but I don't see this happening very often

Reply:
A new release of the model, you mean?

Reply:
Yes